dotnet-datetime/CheckNumberOfMonthsBetweenTwoDates/CheckNumberOfMonthsBetweenTwoDates/NumberOfMonthsBetweenTwoDates.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace CheckNumberOfMonthsBetweenTwoDates;
+
+public static class NumberOfMonthsBetweenTwoDates
+{
+    public static int CalculateSubscriptionDuration(DateTime subscriptionStart, DateTime currentDate)
+    {
+        int months = (currentDate.Year - subscriptionStart.Year) * 12 + currentDate.Month - subscriptionStart.Month;
+
+        if (currentDate.Day < subscriptionStart.Day)
+        {
+            months--;
+        }
+
+        return months;
+    }
+
+    public static double CalculateCourseDuration(DateTime courseStart, DateTime currentDate)
```

Comment:
Suggest changing the name of the second parameter to something like "endDate". Using "currentDate" in the method definition sends the wrong message to users.